db/production--generated.diff

```diff
@@ -0,0 +1,16 @@
+13c13
+< ActiveRecord::Schema.define(version: 2020_05_08_201027) do
+---
+> ActiveRecord::Schema.define(version: 2020_06_17_105905) do
+17d16
+<   enable_extension "pg_stat_statements"
```

Comment:
This was done i *believe* directly in prod for @annaswims

Reply:
I think that's correct. I recall asking someone in ops to do it.   Not that I especially like having differences, but if developers want to enable it locally they have to change their postgres config and restart their db.